### PR TITLE
BeTrains for Nokia, first release in ovi store

### PR DESCRIPTION
**Pull request by Bertware**
First release in Ovi store, fully working, might contain minor bugs, needs code cleanup

### CODE REVIEW

**Comment by maleadt** [comment]
Hi there,

Thanks for the merge request. Some comments:
- Why a single commit? Is this a squashed merge commit? If possible, split into multiple individual commits. This makes the patch way to big to review.
- Executables (.sis files) do not belong in a code repository
- Some files contain local paths (e.g. BeTrains_template.pkg, ABLD.BAT, ...)
- Don't include autogenerated / temporary files (all of the moc/ source-files, .make.cache, ...)
- What is the CppUtils class used for?
- And finally: removing the COPYING and AUTHORS files is simply not done!

TLDR: please clean-up first :-)

**Comment by Bertware** [comment]
Hi,

this is the first commit, as there were no commits during the development, I'll try to commit when I update something in the code.
Authors and copying should be included in the commit, they were not intended to remove and my git reported that they were committed...
i'll clean up the files and remove the executables :)

greetings,

(p.s.: do you speak dutch? And pieter knows about this,too)

-----Origineel bericht-----
Van: Tim Besard
Verz.: 28-02-2013, 08:45
Aan: iRail/BeTrains.QML
CC: Bertware
Ondrw.: Re: [BeTrains.QML] BeTrains for Nokia, first release in ovi store (#1)

Hi there,

Thanks for the merge request. Some comments:
- Why a single commit? Is this a squashed merge commit? If possible, split into multiple individual commits. This makes the patch way to big to review.
- Executables (.sis files) do not belong in a code repository
- Some files contain local paths (e.g. BeTrains_template.pkg, ABLD.BAT, ...)
- Don't include autogenerated / temporary files (all of the moc/ source-files, .make.cache, ...)
- What is the CppUtils class used for?
- And finally: removing the COPYING and AUTHORS files is simply not done!

TLDR: please clean-up first :-)

---

**Comment by pietercolpaert** [comment]
@maleadt any update on this merge? ;)

**Comment by maleadt** [comment]
Huh, that's curious, I completely missed @Bertware's reply / clean-up commit. Shame on me.

There's still some minor things I'd like to see corrected (undelete `.gitignore`, and there's still some `D:/Users/Bert/` paths in the patch), but I'll have a look at it myself later this week.

**Comment by maleadt** [comment]
@pietercolpaert: seems like I don't have push access to the iRail repositories anymore, so I've pushed to my fork instead. Feel free to merge maleadt/BeTrains.QML@3d3da306f31436687bf6280743b7517d553f3ffe, which contains the cleaned-up version of this PR. You can close the PR afterwards, but I'd keep it here for archiving purposes (ie. don't delete it).

**Comment by pietercolpaert** [comment]
@maleadt - gave you access on the repository again. No clue why this was revoked at some point

**Comment by maleadt** [comment]
OK, thanks; pulled & closing this PR.
